tests/resources/Util.robot

```diff
@@ -143,6 +143,7 @@ Cleanup VIC Appliance On Test Server
     Log To Console  Deleting the VCH appliance ${vch-name}
     ${output}=  Run VIC Machine Delete Command
     Run Keyword And Ignore Error  Cleanup VCH Bridge Network  ${vch-name}
+    Run  govc pool.destroy "*/Resources/*"
```

Comment:
Good morning, sorry shud have given you the context on closing the old one. The Util change affected all the other CI test. So wanted to know the best course of action on how to do the clean up.